### ISSUE TITLE
withText usage : ReferenceError: assign is not defined

### ISSUE THREAD

**Issue by Fridus** [synacor/preact-i18n#5]
When I use `withText` with functional components

```
preact-i18n.js:formatted:1 Uncaught (in promise) ReferenceError: assign is not defined
    at new <anonymous> (preact-i18n.js:formatted:1)
    at Component.doRender [as render] (preact-i18n.js:formatted:1)
    at renderComponent (preact-i18n.js:formatted:1)
    at setComponentProps (preact-i18n.js:formatted:1)
    at renderComponent (preact-i18n.js:formatted:1)
    at rerender (preact-i18n.js:formatted:1)
    at <anonymous>
```

```js
  const Aaa = withText('button.next')(() => (<div>Hello</div>))
  render((
    <IntlProvider definition={definition}>
      <Aaa />
    </IntlProvider>
  ), el)
```

If you test with compiled file :
`test/index.js` 
```js
import wrap, { intl, IntlProvider, Text, Localizer, withText } from '../dist/preact-i18n'
```

![capture d ecran 2017-08-03 a 15 17 55](https://user-images.githubusercontent.com/3214565/28923628-fa4b39d4-785e-11e7-9685-35a35ec027b8.png)

**Comment by Fridus** [comment]
Fixed by https://github.com/synacor/preact-i18n/pull/6

**Comment by Fridus** [comment]
cc @developit

**Comment by billneff79** [comment]
Fixed via #7 .  Thanks for the PR @Fridus - good catch!  I'll get a new release out as soon as I can get in touch with @developit

**Comment by billneff79** [comment]
@Fridus Fix is now live on npm as `preact-i18n@1.0.1`

**Comment by Fridus** [comment]
Hi @billneff79 
Thx, happy to read that :)
